### ISSUE TITLE
Examples not working with scalar

### ISSUE THREAD

**Issue by richard-ejem** [APIs-guru/graphql-faker#77]
Schema:
```
type Query {
  date: Date 
}
scalar Date @examples(values: ["2019-08-18"])
```
Query:
```
{
  date
}
```
Expected result:
```
{
  "date": "2019-08-18"
}
```
Actual result:
```
{
  "date": "<Date>"
}
```
Broken in version: 2.0.0-rc.6
Worked properly in 1.9.2

**Comment by IvanGoncharov** [comment]
@richard-ejem Thanks for the detailed report 👍 
Fixed in `2.0.0-rc.7` 🎉 
Can you please test it?

**Comment by richard-ejem** [comment]
It works, thanks 👍

**Comment by nmoore93** [comment]
Hi @IvanGoncharov , thanks very much for this package, it is great! It looks like this issue or something similar with the `@fake` directive may be happening again in version `2.0.0-rc.11`. When I try the following example:

`scalar Date @fake(type: pastDate, options: {dateFormat: "X"})`

Querying for a `Date` field is returning the value `<Date>`

**Comment by IvanGoncharov** [comment]
@nmoore93 Thanks for reporting 👍 
Fixed in `2.0.0-rc.12` 📦 can you please test it?

P.S. Please don't shy to open new issues if you discover stuff like this. It's hard to track reports inside closed issues since if you missed an email from GitHub you will never know about it.

**Comment by nmoore93** [comment]
Thanks very much for the quick reply and fix!
I can confirm that it works for me now in version `2.0.0-rc.12` 👍

I will also keep that in mind about creating new issues, wasn't sure if this new issue was related to this issue, thanks!
